src/base/bittorrent/trackerentry.cpp

```diff
@@ -162,5 +162,6 @@ bool BitTorrent::operator==(const TrackerEntry &left, const TrackerEntry &right)
 
 uint BitTorrent::qHash(const TrackerEntry &key, const uint seed)
 {
-    return (::qHash(key.url(), seed) ^ key.tier());
+    return (::qHash(key.url(), seed)
+        ^ ::qHash(key.tier(), seed));
```

Comment:
Is it too long to split it into two lines?

Reply:
fixed.